src-tauri/Cargo.toml

```diff
@@ -18,6 +18,8 @@ tauri-build = { version = "1.5.0", features = [] }
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 tauri = { version = "1.5.3", features = [ "window-unminimize", "window-unmaximize", "window-maximize", "window-show", "window-minimize", "window-close", "window-start-dragging", "window-hide"] }
+serde_yaml = "0.9.30"
+home = "0.5.9"
```

Comment:
@fersaru: I have noticed, that you already used tauri's home_dir implementation. I switched to a separate crate since I want to keep the `config.rs` module independent from tauri.